### ISSUE TITLE
Ensure tracking service is not killed while in background

### ISSUE THREAD

**Issue by jaranflaath** [agens-no/smittesporing#23]
The background service seems to be killed by the system. Consider excemption form battery optimization?

**Comment by hfossli** [comment]
Read about a possible solution here https://hackernoon.com/android-location-tracking-with-a-service-80940218f561 🎉

**Comment by jaranflaath** [comment]
Yes, this is usually a solution, but device manufacturers like Huawei keep doing their own "clever" battery optimizations and keeps killing off background apps. In for a fun ride!
